src/components/Button/Button.tsx

```diff
@@ -23,52 +23,37 @@ export interface ButtonOptions {
 }
 
 interface ButtonProps {
-    /** ID that other components can cross reference for accessibility purposes */
-    id?: string;
-
-    /** The action to perform on the <button>'s onClick function */
-    onClick: (event: React.MouseEvent | React.KeyboardEvent) => void;
-
     /** Additional attributes passed to the button */
     attributes?: {};
```

Comment:
These aren't used in `button`

Reply:
When you say they're not used in `button`, what do you mean? Meaning L157 isn't doing anything?

Reply:
So weird—I don't see them in Github's highlighted changes, but I am applying them on L157... so weird 🤔

Reply:
wait whaaaaaaaaatttttt

Reply:
:gasp:

Reply:
No me gusta ☹️